acex.tex

```diff
@@ -1376,10 +1384,12 @@ \subsection{Running ACER}
    !
    ! card 11
    !    matd     material to be processed
+   !             photoatomic data on nendf
+   !             atomic relaxation data on npend
    !
    !   --- photo-nuclear data (iopt=5 only) ---
    !
-   ! card 12
+   ! card 11
```

Comment:
We intentionally changed this to `card 12` previously.

Reply:
I actually copied those pieces from the source code so they would be the same. I'll change it as required.